### DISCUSSION TITLE
File Location on Linux Flatpak

### DISCUSSION THREAD

**Discussion by charitarthchugh** [saber-notes/saber#1307] [Q&A]
Can someone tell me what directory saber syncs to?

**Comment by ceskyDJ** [comment]
Hi! What do you mean by syncing?

If it's syncing to Nextcloud instance, Saber syncs to ”NC_DATA/user/files/Saber“ directory (e.g. `/var/www/nextcloud/data/ceskyDJ/files/Saber`), so in web interface of Nextcloud in the Files app you will see it in as ”Saber“ in your (user space) root directory.
